manifests/manage/bzr.pp

```diff
@@ -0,0 +1,18 @@
+# @summary Manage the Bazaar source code manager package
+#
+# @param package_name
+#   name of package to manage
+#
+# @param package_ensure
+#   ensure state of the package resource
+#
+# @example
+#   include vcsrepo::manage::bzr
+class vcsrepo::manage::bzr (
+  Variant[String, Array[String]] $package_name,
+  String $package_ensure,
```

Comment:
can you please use an enum here?

Reply:
Please don't. Package ensures also accept version numbers so `String[1]` really is the closest you can get. There's https://github.com/puppetlabs/puppetlabs-stdlib/pull/1281 open to standardize it and avoid different implementations.

Reply:
I'm not sure that would work if folks wanted to specify a specific version...